modin/experimental/torch/tests/test_dataloader.py

```diff
@@ -0,0 +1,74 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+from __future__ import annotations
+
+from types import ModuleType
+
+import pandas as pd
+from torch.utils.data import RandomSampler
+
+from modin import pandas as mpd
```

Comment:
we write the tests using `import modin.pandas as pd` form, like we say in README for the users; could you please also use that form of import so everything is uniform in the codebase?